assets/js/collaborative-editor/utils/workflowGraph.ts

```diff
@@ -518,3 +518,49 @@ export function findFirstJobFromTrigger(
   const edge = edges.find(e => e.source_trigger_id === triggerId);
   return edge?.target_job_id || undefined;
 }
+
+// gets job ordinals relative to start(trigger-node)
+export function getJobOrdinals(adj: Record<string, string[]>, start: string) {
```

Comment:
Should we add the return type. Something like `getJobOrdinals(adj: Record<string, string[]>, start: string): Record<string, number>` ?